src/commcare_cloud/ansible/roles/commcarehq/templates/supervisor_django.conf.j2

```diff
@@ -0,0 +1,16 @@
+[program:{{ project }}-{{ deploy_env }}-django]
+directory={{ code_home }}/
+; gunicorn
+environment={% for name, value in item.env_vars.items() %}{{ name }}="{{ value }}",{% endfor %}
+
```

Comment:
This extra line is necessary because ansible appears to be not add a line ending on line 4 after the for loop. not sure whyt hat is